docs/epidata_development.md

```diff
@@ -49,346 +49,125 @@ $ [sudo] make test pdb=1
 $ [sudo] make test test=repos/delphi/delphi-epidata/integrations/acquisition
 ```
 
-## Long version
+You can read the commands executed by the Makefile [here](../dev/local/Makefile).
 
-**Prerequisite:** this guide assumes that you have read the
-[frontend development guide](https://github.com/cmu-delphi/operations/blob/main/docs/frontend_development.md).
+## Rapid Iteration and Bind Mounts
 
-This guide describes how to write and test code for the Epidata API. For
-preliminary steps,
-[install docker and create a virtual network](https://github.com/cmu-delphi/operations/blob/main/docs/frontend_development.md#setup).
+To reduce friction, we
+[bind-mount](https://docs.docker.com/storage/bind-mounts/) local source files to
+the containers, which replaces the corresponding files from the image and allows
+your code changes to be reflected immediately, without needing to rebuild. This
+approach comes with some drawbacks you should be aware of:
 
-After reading this guide, you may want to visit
-[the `fluview_meta` tutorial](new_endpoint_tutorial.md) for an example of how
-to add a new endpoint to the API.
+- the container will be able read and write to your local filesystem (which may
+  be a security concern, especially if you are running the containers as root)
```

Comment:
dont they always run as root?

Reply:
not always https://docs.docker.com/engine/security/rootless/